systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -390,6 +396,97 @@ void testTlsExternalUser(ExtensionContext extensionContext) {
         assertThat(user.getStatus().getUsername(), is("CN=" + userName));
     }
 
+    /**
+     * @description This test case checks that KafkaUser custom resource is managed correctly, even in presence of more than just one User Operator.
+     *
+     * @steps
+     *  1. - In addition to already existing Kafka Cluster (A), deploy another Kafka Cluster (B) in the same Namespace
+     *     - Kafka (B) is deployed
+     *  2. - Create KafkaUser custom resource with metadata due to which it will be listened to by User Operator belonging to the Kafka Cluster B
+     *     - Secret with label referencing Kafka Cluster B is created, and Kafka resource user corresponding to created KafkaUser custom resource exists only in Kafka Cluster B
+     *
+     * @usecase
+     *  - labels
+     *  - user-operator
+     *  - users
+     *  - secrets
+     */
+    @ParallelTest
+    @KRaftNotSupported("Probably bug in Kafka - https://issues.apache.org/jira/browse/KAFKA-13964")
```

Comment:
The issue is resolved, could you give it a try in KRaft mode?

Reply:
true, this change (resolution) will actually take place from kafka version 3.5.0 onwards.